Makefile

```diff
@@ -149,7 +149,6 @@ ifeq ($(ENV_ARG), dev)
 	@$(BASH_DEV) "RAILS_ENV=test DISABLE_BOOTSNAP=true NOCOVERAGE=true bundle exec parallel_rspec ${SPEC_PATH}"
 else
 	@$(COMPOSE_TEST) $(BASH) -c "CIRCLE_JOB=true RAILS_ENV=test DISABLE_BOOTSNAP=true bundle exec parallel_rspec ${SPEC_PATH}"
-	@$(BASH_TEST) "DISABLE_BOOTSNAP=true RUN_COVERAGE=true bundle exec rake simplecov:report_coverage"
```

Comment:
Should this be removed?

Reply:
Oh, so it looks like if we're not using this anymore, can we just completely remove simplecov_parallel.rake and the simplecov_helper

Reply:
From what I could tell it wasn't doing anything. That rake task is specific to circleci file path for coverage and cirlceci doesn't use the this makefile only Jenkins does. 

CircleCI reports coverage here: https://github.com/department-of-veterans-affairs/vets-api/blob/master/.circleci/config.yml#L111  

Jenkins does it here: https://github.com/department-of-veterans-affairs/vets-api/blob/master/Jenkinsfile#L81-L84

Reply:
CircleCI still uses `simplecov_parallel.rake` and the `simplecov_helper`

Reply:
Ah, ok! Makes sense!